src/OrchardCore.Modules/OrchardCore.OpenId/Startup.cs

```diff
@@ -73,19 +73,14 @@ public override void ConfigureServices(IServiceCollection services)
                 ServiceDescriptor.Scoped<INavigationProvider, AdminMenu>(),
             });
 
-            services.Configure<CookiePolicyOptions>(options =>
+            services.ConfigureApplicationCookie(options =>
             {
-                options.OnAppendCookie = cookieContext =>
-                {
-                    // Disabling same-site is required for OpenID's module prompt=none support to work correctly.
-                    // Note: it has no practical impact on the security of the site since all endpoints are always
-                    // protected by antiforgery checks, that are enforced with or without this setting being changed.
-                    // 2020-03-23; Moved the SameSiteNode.None here, this will require that the site runs on HTTPS;
-                    if (cookieContext.CookieName.StartsWith("orchauth_"))
```

Comment:
@deanmarcussen I'm curious to know if this has any other implications ? This one was disabling the SameSiteMode only for the orcauth_ cookies

Reply:
For me I don't think this changes anything. It is still configuring the application cookie, it's just done at the time the cookie is generated, rather than doing it later, in the `CookieMiddleware`

/cc @MatthijsKrempel